scripts/windows_compile.bat

```diff
@@ -152,23 +149,19 @@ ECHO Creating the project with flags: %FLAGS%
 
 if %DEBUG_BUILD% == T (
   ECHO Generating Debug Project
-  qmake -tp vc src/src.pro CONFIG+=debug %FLAGS%
+  qmake src/src.pro CONFIG+=debug %FLAGS%
```

Comment:
why do we need to change how we compile the project? I'm OK with this, I just need to know more.

Reply:
We should leave the -tp vc in.  For local developers this switch is what generates the Visual Studio project.

Reply:
We can't use msvc on the tc-windows vm as it's missing some .net dependency. (I can't resolve it, as we don't have admin) 
How about -vc as default and having a switch for nmake?